samples/cpp/stitching_detailed.cpp

```diff
@@ -813,7 +817,7 @@ int main(int argc, char* argv[])
             }
             blender->prepare(corners, sizes);
         }
-        else if (!timelapser)
+        else if (!timelapser && timelapse)
```

Comment:
This sample is rather old and big (~1k lines). A log of changes have been added here (includes possible caching for Blender object, which looks curiously now).
I believe it will be good to have separate PRs with new changes (this PR already has many fixes)